forte/data/data_store.py

```diff
@@ -160,9 +161,16 @@ def __init__(
         ``type_name``, their parent entry, and the order of corresponding attributes.
         The keys are fully qualified names of every type; The value is a
         dictionary with two keys. Key ``attribute`` provides an inner dictionary
-        with all valid attributes for this type and the indices of attributes
-        among these lists. Key ``parent_class`` is a string representing the
-        ancestors of this type.
+        with all valid attributes for this type and the information of attributes
+        among these lists. This information is represented as a dictionary. The
+        dictionary has two entries, the first is index which determines the position
+        where an attribute is stored in a data store entry. The second is type, which
+        is a tuple of two elements that provides the type information of a given
+        attribute. The first element is the unsubscripted version of
+        the attribute's type and the second element is the type arguments
```

Comment:
examples will help this a lot

Reply:
The example is given below

Reply:
it is pretty hard to understand this section now

Reply:
Oh, I'll try explaining it better. Is there any area specifically that you feel needs to change?